### PR TITLE
Update Nuget for Bot and SDK.

### PR DESCRIPTION
**Pull request by MikeStall**
SDK has a breaking change for Bot, so we need a new Bot nuget too.

### CODE REVIEW

File: src/WebJobs.Script.WebHost/Web.config [hunk 1]
```diff
@@ -181,7 +181,7 @@
       <dependentAssembly>
         <assemblyIdentity name="System.IO.Compression" publicKeyToken="b77a5c561934e089" culture="neutral" />
         <!--Explicitly redirecting back to 4.0.0. See https://github.com/dotnet/corefx/issues/15693 for details-->
-        <bindingRedirect oldVersion="0.0.0.0-4.1.2.0" newVersion="4.0.0.0" />
```
Comment (mathewc) [inline src/WebJobs.Script.WebHost/Web.config]: You have to revert this file

File: src/WebJobs.Script/Host/ScriptHost.cs [hunk 2]
```diff
@@ -410,8 +409,13 @@ protected virtual void Initialize()
                 }
 
                 // Load builtin extensions
-                var sendGridExtension = new Extensions.SendGrid.SendGridConfiguration();
-                LoadExtension(sendGridExtension);
+                {
+                    var botExtension = new Extensions.BotFramework.Config.BotFrameworkConfiguration();
```
Comment (mathewc) [inline src/WebJobs.Script/Host/ScriptHost.cs]: Have you made sure all the host level config knobs on BotFrameworkConfiguration still are settable via host.json after the replat?

Comment (MikeStall) [inline src/WebJobs.Script/Host/ScriptHost.cs]: Yes. They have unit tests, which I've verified. This happens in the new ApplyConfig()  in their extension.

File: src/WebJobs.Script.WebHost/Web.config (hunk 1 above)
Comment (MikeStall) [inline src/WebJobs.Script.WebHost/Web.config]: d'oh. gets me every time. Is there some test we can add for this? Do an xpath on the XML file?

**Comment by fabiocav** [comment]
@MikeStall , this has been merged, correct?

**Comment by MikeStall** [comment]
Merged

**Pull request closed without merging**